CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 - **Breaking:** On Web, `set_cursor_position` and `set_cursor_grab` will now always return an error.
 - **Breaking:** `PixelDelta` scroll events now return a `PhysicalPosition`.
 - On NetBSD, fixed crash due to incorrect detection of the main thread.
+- **Breaking:** The virtual key code `Subtract` has been renamed to `NumpadSubtract`
+- **Breaking:** On Linux, `-` key is mapped to the `Minus` virtual key code, instead of `Subtract`
```

Comment:
It should be `On X11`. There's no `linux` in particular when it comes to winit, and Wayland was fine.